core/views.py

```diff
@@ -342,6 +342,12 @@ def get_context_data(self: "CommunityDetailView", **kwargs: any) -> dict[str, an
         context["moderators"] = CommunityMember.objects.filter(
             community=community, role=CommunityMember.MODERATOR
         ).select_related("user")
+        context["community_avatar_url"] = (
+            community.avatar.url if community.avatar else "/media/community_avatars/default.jpg"
+        )
+        context["community_background_url"] = (
```

Comment:
I think community class should handle this itself. Please move to community class logic.